docs/articles/documentation/test-api/selecting-page-elements/README.md

```diff
@@ -20,3 +20,4 @@ use it in actions and assertions.
 * [Selectors](selectors/README.md)
 * [DOM Node State](dom-node-state.md)
 * [Framework-Specific Selectors](framework-specific-selectors.md)
+* [Examples of Working with Page DOM](examples-of-working-with-page-dom.md)
```

Comment:
I don't like this title `Examples of Working with Page DOM`.
`DOM` is a very broad concept. It contains elements, event, storage and etc.
You example covers only elements and their properties.

Reply:
I rather agree with @miherlosev here. Maybe `... with DOM elements`?
Do you have any other proposals?

Reply:
hmm... I think that `with DOM elements` looks okay to me